helpers/flex.css

```diff
@@ -0,0 +1,8 @@
+.flex {
```

Comment:
Don't make different css files for these utility classes just put them in one file !!

Reply:
got it, thanks @Priya-Patel31